techmatters/flex-plugins
# CHI-2225 Align case related dates to show user's browser/machine time timezones

Primary reviewer:

## Description
- This PR align all the case related dates to show the correct date based on the local browser/machine time zone. 
- Remove the use of `getLocaleDateTime` helper function to `parseISO(inputDate).toLocalDateString` 
- Followup ticket has been created [to consolidate date formatting code](https://tech-matters.atlassian.net/browse/CHI-2586?atlOrigin=eyJpIjoiYzY3NWE3NDYxMWVmNDI5ZThiNzA5MmM0OTI2MzE1MTQiLCJwIjoiamlyYS1zbGFjay1pbnQifQ) See [related slack thread here](https://tech-matters.slack.com/archives/C05JDH53S65/p1707428720148229?thread_ts=1707428254.633059&cid=C05JDH53S65)

### Checklist

### Related Issues
Fixes # https://tech-matters.atlassian.net/browse/CHI-2225

### Verification steps
1. Install the "Spoof Timezone" extension from the following URL: https://webextension.org/listing/spoof-timezone.html. Ensure the extension is updated based on the VPN's new IP address.
2. Create a new case
3. Verify the consistency of the case's date across various displays. Specifically, ensure that the date shown in the table row listing the case, the detailed view of the case, and the case preview within search results are all identical.
4. Reset vpn and browser to your current timezone. Check the same displays.
